p2pool/dash/networks/dash.py

```diff
@@ -13,16 +13,16 @@
 SCRIPT_ADDRESS_VERSION = 16
 RPC_PORT = 9998
 RPC_CHECK = defer.inlineCallbacks(lambda dashd: defer.returnValue(
-            'dash' in (yield dashd.rpc_help()) and
+            (yield helper.check_block_header(dashd, '000007d91d1254d60e2dd1ae580383070a4ddffa4c64c2eeb4a2f9ecc0414343')) and
```

Comment:
```suggestion
            (yield helper.check_block_header(dashd, '00000ffd590b1485b3caadc19b22e6379c733355108f107a430458cdf3407ab6')) and
```
https://github.com/dashpay/dash/blob/master/src/chainparams.cpp#L292